hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/resolver/MountTableResolver.java

```diff
@@ -97,6 +99,8 @@
   private final TreeMap<String, MountTable> tree = new TreeMap<>();
   /** Path -> Remote location. */
   private final Cache<String, PathLocation> locationCache;
+  public static LongAdder locCacheMissed = new LongAdder();
+  public static LongAdder locCacheAccessed = new LongAdder();
```

Comment:
Those metrics make sense to me. 
One little advice: update `locCacheMissed` to `locCacheMiss`, and update `locCacheAccessed` to `locCacheHit`.

Reply:
Please also fix the spotbugs. Thanks.

Reply:
> 

Thanks @tomscut for reviewing the code.  I have updated locCacheMissed to  locCacheMiss. But the field locCacheAccessed could not update to locCacheHit. beacuase it mean total cache access count, not cache hit count.
Finally, we compute hit rate use : locCacheMiss/locCacheAccess

Reply:
> 

OK, I will fix it.

Reply:
> > 
> 
> Thanks @tomscut for reviewing the code. I have updated locCacheMissed to locCacheMiss. But the field locCacheAccessed could not update to locCacheHit. beacuase it mean total cache access count, not cache hit count. Finally, we compute hit rate use : locCacheMiss/locCacheAccess

I see. Thanks.